### PR TITLE
added ipv6 clarification.

### PR DESCRIPTION
**Pull request by siennathesane**
made things a bit more clear.

Signed-off-by: Mike Lloyd <mike.lloyd@gsa.gov>

//cc @bengerman13

### CODE REVIEW

**Review by bengerman13 — DISMISSED** [review]
:shipit:

**Review by mogul — APPROVED** [review]

**Comment by mogul** [comment]
@pburkholder FYI... Suggest changes if you think this isn't clear enough.

**Comment by siennathesane** [comment]
//cc @pburkholder

File: content/docs/compliance/domain-standards.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Here's what cloud.gov does to support relevant federal standards and recommendat
...
+cloud.gov has basic support for IPv6. cloud.gov supports two types of application access, external and internal. External access is when traffic reaches an application from outside the cloud.gov platform, such as from an office, external application, or user. Internal traffic is traffic which leverages the platform's internal DNS to allow applications to communicate without ever leaving the platform's network boundaries. IPv6 is supported for external access, but not internal access, only IPv4 is supported. When you deploy a new application, for external IPv6 traffic, you do not have to take any action, and most applications will know to use IPv4 internally as cloud.gov's internal DNS provider will leverage IPv4. If you have more questions or security questions, please contact [Support]((mailto:cloud-gov-support@gsa.gov).
```
Comment (mogul) [inline content/docs/compliance/domain-standards.md]: Should probably clean up "but not internal access, only IPv4 is supported" to "but for internal access, only IPv4 is supported" to remove ambiguity.

Comment (mogul) [reply content/docs/compliance/domain-standards.md]: (Sorry, shoulda put that in a review and requested changes.)

**Review by mogul — CHANGES_REQUESTED** [review]
Just reopening to make sure the point above gets addressed.

**Review by mogul — APPROVED** [review]

**Pull request merged**